src/main/java/com/kainos/ea/backend/services/JobFamilyService.java

```diff
@@ -21,4 +22,7 @@ public JobFamilyService(JobFamilyRepository jobFamilyRepository) {
         return jobFamilyRepository.findByCapabilityName(capabilityName);
     }
 
+    public void deleteJobFamily(String jobFamilyName) throws EmptyResultDataAccessException {
+        jobFamilyRepository.deleteById("");
```

Comment:
Shouldn't it be *deleteById(jobFamilyName);**?

Reply:
Yep, you're right, I must have changed it by accident when writing tests